servo/src/server/routes/mappings.rs

```diff
@@ -18,6 +16,261 @@ pub struct GetMappingResponse {
   pub mappings: Vec<NodeMapping>,
 }
 
+const PYTHON_KEYWORDS : [&str; 70] = [
+  "abs", "aiter", "all", "anext", "any", "ascii", "bin", "bool", "breakpoint", 
+  "bytearray", "bytes", "callable", "chr", "classmethod", "compile", "complex", 
+  "delattr", "dict", "dir", "divmod", "enumerate", "eval", "exec", "filter", 
+  "float", "format", "frozenset", "getattr", "globals", "hasattr", "hash", 
+  "help", "hex", "id", "input", "int", "isinstance", "issubclass", "iter", 
+  "len", "list", "locals", "map", "max", "memoryview", "min", "next", "object", 
+  "oct", "open", "ord", "pow", "print", "property", "range", "repr", "reversed",
+  "round", "set", "setattr", "slice", "sorted", "staticmethod", "str", "sum", 
+  "super", "tuple", "type", "vars", "zip"
+];
+    
+/// Returns where an identifier is a preexisting python keyword
+/// (Only checks base python functions)
+fn is_python_keyword(identifier : &String) -> bool {
```

Comment:
This function should take a `&str`. `String` can deref into a `str`. Also, formatting.